Sphinx version

After merging PR #123, sphinx (1.2.3) gave me an error when building the doc:
`sphinx.errors.ThemeError: no theme named 'classic' found (missing theme.conf?)`

I fixed it by upgrading my sphinx to 1.4.4. 

I can see the same problem on the nightly build log. Maybe a sphinx upgrade is necessary on the server.

Actually, it is event a FBFS on debian 8
 
sphinx.errors.ThemeError: no theme named 'classic' found (missing theme.conf?)
debian/rules:25: recipe for target 'override_dh_auto_build' failed
make[1]: *** [override_dh_auto_build] Error 1
make[1]: Leaving directory '/home/kieffer/workspace/silx/package/silx-0.2.0a0'
debian/rules:13: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
Build failed, please investigate ...

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -104,7 +104,7 @@ pygments_style = 'sphinx'
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'classic'
+html_theme = 'default'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the

